contracts/EscrowNative.sol

```diff
@@ -53,7 +53,7 @@ contract EscrowNative is IEscrow {
 
         require(msg.value == paymentAmount + referralTax + platformTax, "Invalid amount sent");
 
-        IRewarder(rewarder).handleSeekerTaxNative{ value: referralTax + platformTax }(_seekerId, referralTax, platformTax);
+        IRewarder(rewarder).handleSeekerTaxNative{ value: referralTax + platformTax }(_seekerId, _solverId, referralTax, platformTax);
```

Comment:
why do we need solver id when taxing seeker?

Reply:
Changes reverted